src/pages/resources/testbeds.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import styled from 'styled-components'
+import { useTestbeds } from '../../hooks'
+import { AnimateOnMount } from '../../components/anim'
+import { SEO } from '../../components/seo'
+import { Subheading, Title, Paragraph } from '../../components/typography'
+import { LinkIcon } from "../../components/icons"
+import { Container as Grid, Row, Col } from 'react-grid-system'
+
+const TestbedWrapper = styled.article`
+  height: 600px;
+  margin: 0 0 2rem 0;
+  background-color: var(--color-lightgrey);
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  border-radius: 4px;
+  transition: filter 250ms;
+  filter: drop-shadow(0 0 3px #00000033);
+  &:hover {
+    filter: drop-shadow(0 0 5px #00000033);
+  }
+  & .header {
+    min-height: 200px;
+    max-height: 200px;
+    border: 1px dashed crimson;
+    background-color: var(--color-white);
+    border: solid var(--color-primary);
+    border-width: 0 0 4px 0;
+    padding: 1rem;
+    background-repeat: no-repeat;
+    background-size: 150px;
+    background-position: center center;
+  }
+  & .description {
+    flex: 1;
+    padding: 1rem;
+    overflow: auto;
+    // color: var(--color-white);
+  }
+  & .footer {
+    display: flex;
+    justify-content: flex-end;
+    background-color: var(--color-grey);
+    & .testbed-link {
+      width: 3rem;
+      height: 3rem;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+  }
+`
+
+const TestbedCard = ({ testbed }) => {
+  return (
+    <TestbedWrapper>
+      <div className="header" style={{ backgroundImage: `url(${ testbed.image.childImageSharp.original.src })` }}>
+        <Subheading center hidden>{ testbed.name }</Subheading>
+      </div>
+      <div className="description">
+        <Paragraph>
+          { testbed.description }
+        </Paragraph>
+      </div>
+      <div className="footer">
+        <a href={ testbed.url } rel="noopener noreferrer" className="testbed-link" aria-label={ `Visit the ${ testbed.name } website` }>
```

Comment:
Would adding "target=_blank" to the <a> be better? In this way users can open multiple testbeds' website with multiple tabs easier.

Reply:
Also, is it possible to make the whole footer clickable as the link? I was confused at first by clicking the footer but not being directed to a new page. The footer changes color by hovering on it, which may make users think the whole footer is clickable as a link button.

Reply:
yep, i totally forgot the target attr. nice catch! regarding the footer, i agree. does this new implementation feel right?

Reply:
Yes the new implementation looks great!